config.json

```diff
@@ -8,7 +8,7 @@
     "representer": false,
     "analyzer": false
   },
-  "blurb": "TODO: add blurb",
+  "blurb": "Batch was a scripting language which is one of the best languages with backward support, it is still commonly used today because of its unorthodox dynamic structure.",
```

Comment:
Maybe tweak to:

```suggestion
  "blurb": "Batch is a scripting language with great backwards compatibility. It is still commonly used today because of its unorthodox dynamic structure.",
```

Reply:
Yeah, that's better, thanks you sir.